src/selectors/linesInScope.js

```diff
@@ -0,0 +1,41 @@
+import { getOutOfScopeLocations, getSelectedSource } from "../selectors";
+
+import range from "lodash/range";
+import flatMap from "lodash/flatMap";
+import uniq from "lodash/uniq";
+import without from "lodash/without";
+
+function getLinesOutOfScope(outOfScopeLocations: AstLocation[]) {
+  if (!outOfScopeLocations) {
```

Comment:
if this is an array of AstLocations, I think this will always be true... or maybe it isnt an array?

if it is an array we probably need to do 
```javascript
if (!outOfScopeLocations.length) {
  ...
}
```